addons/event/views/event_ticket_views.xml

```diff
@@ -28,4 +28,96 @@
 		</field>
 	</record>
 
+    <!-- EVENT.TICKET -->
+    <record id="event_event_ticket_view_tree_from_event" model="ir.ui.view">
+        <field name="name">event.event.ticket.view.tree.from.event</field>
+        <field name="model">event.event.ticket</field>
+        <field name="priority" eval="20"/>
+        <field name="arch" type="xml">
+            <tree string="Tickets" editable="bottom">
+                <field name="name"/>
+                <field name="start_sale_date"/>
+                <field name="end_sale_date"/>
+                <field name="seats_max"/>
+                <field name="seats_reserved" readonly="1"/>
+                <field name="seats_unconfirmed" readonly="1"/>
+            </tree>
+        </field>
+    </record>
+
+    <record id="event_event_ticket_view_form_from_event" model="ir.ui.view">
+        <field name="name">event.event.ticket.view.form.from.event</field>
+        <field name="model">event.event.ticket</field>
+        <field name="priority" eval="20"/>
+        <field name="arch" type="xml">
+            <form string="Ticket">
+                <group>
+                    <group>
+                        <field name="name"/>
+                        <field name="end_sale_date"/>
+                    </group><group>
+                        <field name="seats_max"/>
+                        <field name="seats_reserved" readonly="1"/>
+                        <field name="seats_unconfirmed" readonly="1"/>
+                    </group>
+                </group>
+            </form>
+        </field>
+    </record>
+
+    <record id="event_event_ticket_view_kanban_from_event" model="ir.ui.view">
+        <field name="name">event.event.ticket.view.kanban.from.event</field>
+        <field name="model">event.event.ticket</field>
+        <field name="priority" eval="20"/>
+        <field name="arch" type="xml">
+            <kanban class="o_kanban_mobile">
+                <field name="name"/>
+                <field name="seats_max"/>
+                <field name="seats_reserved"/>
+                <field name="seats_unconfirmed"/>
+                <templates>
+                    <t t-name="kanban-box">
+                        <div t-attf-class="oe_kanban_card oe_kanban_global_click">
+                            <div class="row">
+                                <div class="col-8">
+                                    <strong><t t-esc="record.name.value"/></strong>
+                                </div>
+                            </div>
+                            <div><i>
+                            <t t-esc="record.seats_reserved.value"/> reserved + <t t-esc="record.seats_reserved.value"/> unconfirmed
+                            </i></div>
+                        </div>
+                    </t>
+                </templates>
+            </kanban>
+        </field>
+    </record>
+
+    <record id="event_event_ticket_form_view" model="ir.ui.view">
```

Comment:
So this form view is actually never used/displayed anywhere.
Since we use the tree view in editable=bottom mode.

Do we keep it because of best practice? In case we actually want to use it at some point?
Seems like additional non-required maintenance though.

Reply:
I added menus in config (group no one) for ticket templates and tickets.